_posts/07-04-01-Interacting-via-Code.md

```diff
@@ -95,10 +95,6 @@ This is essentially the same as what most modern frameworks are doing, albeit a
 need to do all of that every time, but mixing together too much presentation logic and database interaction can be a
 real problem if you ever want to [unit-test](/#unit-testing) your application.
 
-[PHPBridge] has a great resource called [Creating a Data Class] which covers a very similar topic, and is great for
-developers just getting used to the concept of interacting with databases.
-
 
 [MVC]: https://code.tutsplus.com/tutorials/mvc-for-noobs--net-10488
-[PHPBridge]: https://phpbridge.org/docs/
```

Comment:
Uh oh... what about the link on the next line ?

Reply:
good catch @jrfnl

Reply:
You are right @jrfnl , sorry my fault. Thanks!